### ISSUE TITLE
JetBrains IDE options to open file links (PhpStorm, WebStorm, ...)

### ISSUE THREAD

**Issue by KevinRousseau** [warpdotdev/Warp#3332]
### Describe the solution you'd like?

```Text
In the settings > features, you have the option to choose an editor to open file links. At the moment you don't have the option to pick a JetBrains IDE such as PhpStorm, WebStorm, ...
```

### Is your feature request related to a problem? Please describe.

```Text
I'm mainly using JetBrains IDE and not having the option to open files from here kind of breaks the development flow for me. 

To give an example:
I mainly work with Laravel (PHP), it has a lot of CLI commands built in to generate files. After creation of the files you get pointed to it with a link. I would then want to just click on it and to open the file in my desired IDE. At the moment this isn't possible for JetBrains IDE.
```

### Additional context

JetBrains IDE don't show up here:

<img width="697" alt="Screenshot 2023-07-08 at 14 37 42" src="https://github.com/warpdotdev/Warp/assets/9538151/394a9627-8ced-40e8-acfc-8c5c6fbc3ad8">

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by KevinRousseau** [comment]
@ElBi21 Glad to see it working with other people! I was in disbelieve they wouldn't support JetBrains IDEs since they're very popular. 

I've rebooted a couple of times since posting. I'm all updated on both PhpStorm (2023.1.3) and Warp (v0.2023.07.04.08.03.stable_02) but still no luck. Something else I might overlook?

**Comment by dannyneira** [comment]
Please check our docs on how you can open jetbrains ide from warp https://docs.warp.dev/features/integrations-and-plugins#jetbrains-ides

I'd check to make sure that Warp has full disk permissions and if you're still not seeing it as an option make sure you opened the ide and completely finished the initial setup. last resort you can try restarting your warp settings completely with `defaults delete dev.warp.Warp-Stable`.

**Comment by KevinRousseau** [comment]
@dannyneira Thank you for your input. I bumped into this article, however it's not what I'm looking for. I'm looking to open files from Warp to PhpStorm, not the other way around.

I gave Warp full disk permissions on my mac and after a reboot of both Warp, and then my mac, nothing changes the situation. 

As @ElBi21 also mentions, I should probably try a reinstall. However this isn't very user friendly. Still love Warp tho :)
